app/templates/press-releases/press-release/edit.hbs

```diff
@@ -0,0 +1,130 @@
+<Auk::Navbar @skin="gray-100" @size="large">
```

Comment:
De navbar kan in de template van de parent-route gezet worden zodanig dat deze gedeeld is. Het is dan voldoende om een `{{#unless this.pressRelease.publication.isPublished}}` te zetten rond de buttons/menus die niet getoond moeten worden als het persbericht al gepubliceerd is. Op die manier vermijden we te veel duplicate code in de template.